### PR TITLE
Updated key and value pairs and templating.

### PR DESCRIPTION
**Pull request by Aura6**

### CODE REVIEW

File: charts/tornjak/values.yaml [hunk 1]
```diff
@@ -12,17 +12,15 @@ clustername: spire-example
...
+# spireVersion: 1.3.5
 
 # SPIRE Server configuration
 spireServer:
   # tornjakImage - Tornjak with SPIRE Server
-  # TODO this is just a temporary image with several patches. It
-  # should be removed after the patches are available in the SPIRE main
-  # img: ghcr.io/spiffe/tornjak-spire-server
-  img: tsidentity/local-spire-server
+  img: ghcr.io/spiffe/tornjak-spire-server
```
Comment (maia-iyer) [inline charts/tornjak/values.yaml:23]: we will soon need to change this image name to `ghcr.io/spiffe/spire-server-tornjak-be` once the latest tornjak prs come out

File: charts/tornjak/values.yaml [hunk 2]
```diff
@@ -80,3 +78,9 @@ attestors:
   azure_msi:
   #   tenants:
   #   - tenant: "TENANT_ID"
+
+tornjak:
+   config:
+     enableUserMgment: true
+     authServerUri: "http://keycloak.tornjak.appdomain.cloud"
+     apiServerUri: "http://tornjak-http-tornjak.appdomain.cloud"
```
Comment (maia-iyer) [inline charts/tornjak/values.yaml:86]: this maybe should be frontend-config? see server-statefulset.tpl comments?

File: charts/tornjak/templates/server-statefulset.tpl [hunk 3]
```diff
@@ -17,148 +36,183 @@ spec:
...
+          readOnly: true
+        - name: spire-oidc-socket
+          mountPath: {{ .Values.oidc.socketDir }}
+      {{- end }}
+      {{- if .Values.tornjak }}
+      {{- if .Values.tornjak.config }}
+      {{- if .Values.tornjak.config.enableUserMgment }}
+      - name: frontend
+        # image: mohammedmunirabdi/deploy-tornjak-frontend-kubernetes:latest
+        # image: tsidentity/tornjak-spire-server:latest
```
Comment (maia-iyer) [inline charts/tornjak/templates/server-statefulset.tpl:165]: dead code?

File: charts/tornjak/templates/server-statefulset.tpl [hunk 4]
```diff
@@ -1,3 +1,22 @@
+{{- if .Values.tornjak }}
+{{- if .Values.tornjak.config }}
+{{- if .Values.tornjak.config.enableUserMgment }}
```
Comment (maia-iyer) [inline charts/tornjak/templates/server-statefulset.tpl:3]: maybe would be good to rename to enableFrontend, which should be separate from enableUserMgment (this is toggled when the authUri is set

**Review by maia-iyer — APPROVED** [review]
overall lgtm, the major comment is that the enableUsrMgmt value should instead be enableFrontend? because UsrMgmt is actually dictated to frontend with the URI given.

File: charts/tornjak/values.yaml (hunk 1 above)
Comment (mrsabath) [reply charts/tornjak/values.yaml:23]: We should change the name once the new image is available. Separate PR.

File: charts/tornjak/values.yaml (hunk 2 above)
Comment (mrsabath) [reply charts/tornjak/values.yaml:86]: I like this idea. I will re-shuffle the configurations. Thank you

**Review by maia-iyer — APPROVED** [review]
lgtm!

**Review by maia-iyer — APPROVED** [review]
lgtm!

**Pull request merged**